src/components/content-tab-assistant.tsx

```diff
@@ -23,25 +24,129 @@ interface MessageProps {
 	className?: string;
 }
 
-export const Message = ( { children, isUser, className }: MessageProps ) => (
-	<div className={ cx( 'flex mt-4', isUser ? 'justify-end' : 'justify-start', className ) }>
-		<div
-			className={ cx(
-				'inline-block p-3 rounded-sm border border-gray-300 lg:max-w-[70%] select-text whitespace-pre-wrap',
-				! isUser && 'bg-white'
-			) }
-		>
-			{ typeof children === 'string' ? (
-				<div id="assistant-markdown">
-					<Markdown>{ children }</Markdown>
-				</div>
-			) : (
-				children
-			) }
+interface InlineCLIProps {
+	output: string;
+	status: 'success' | 'error';
+	time: string;
+}
+
+const InlineCLI = ( { output, status, time }: InlineCLIProps ) => (
+	<div className="mt-1 p-4 bg-[#2D3337]">
+		<div className="flex justify-between mb-2 font-sans">
+			<span className={ status === 'success' ? 'text-[#63CE68]' : 'text-[#E66D6C]' }>
+				{ status === 'success' ? 'Success' : 'Error' }
```

Comment:
Likely no need to repeat the `status` logic here once integrated with wp-cli -- this logic is just for demonstration, and could be refactored into separate states.